tests/libres_tests/res/enkf/test_row_scaling_case.py

```diff
@@ -404,7 +405,7 @@ def test_2ministep(self):
             # The first smoother update without row scaling
             es_update = ESUpdate(main)
             run_context = ErtRunContext.ensemble_smoother_update(init_fs, update_fs1)
-            rng = main.rng()
+            main.rng()
```

Comment:
Needed?

Reply:
At least it verifies that we are able to dive into c-code in this particular context..